app/controllers/dashboard/dashboard-activity-controller.js

```diff
@@ -9,13 +9,15 @@ const moment = require('moment-timezone')
 const response = require('../../utils/response').response
 const CORRELATION_HEADER = require('../../utils/correlation_header').CORRELATION_HEADER
 const ConnectorClient = require('../../services/clients/connector_client').ConnectorClient
+const directDebitConnectorClient = require('../../services/clients/direct_debit_connector_client.js')
```

Comment:
Might be clearer to use destructuring assignment here as in:
`const {DIRECT_DEBIT_TOKEN_PREFIX} = require('../../services/clients/direct_debit_connector_client.js')`